bigquery_etl/cli/query.py

```diff
@@ -1106,5 +1117,18 @@ def validate_schema(name, sql_dir, project_id):
 
     with Pool(8) as p:
         result = p.map(_validate_schema, query_files, chunksize=1)
-    if not all(result):
+
+    all_valid = True
+
+    for r in result:
+        if r[0] is False:
+            if all_valid:
+                click.echo("\nSchemas for the following queries are invalid:")
+
+            all_valid = False
+            click.echo(r[1])
+
+    if not all_valid:
         sys.exit(1)
+    else:
+        click.echo("\nAll schemas are valid.")
```

Comment:
Since the schema validation step potentially logs out a lot of information (e.g. fields where descriptions differ etc.) it could become hard to find the schemas that don't pass validation. So it might be handy to list schemas that failed once validation is complete.